<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>barrawindow</class>
 <widget class="QMainWindow" name="barrawindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QListWidget" name="listWidget_7"/>
     </item>
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget_1"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QListWidget" name="listWidget_4"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QListWidget" name="listWidget_6"/>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QListWidget" name="listWidget_2"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Finalizar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QListWidget" name="listWidget_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QListWidget" name="listWidget_5"/>
     </item>
     <item row="3" column="3">
      <widget class="QListWidget" name="listWidget_8"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
